src/components/TooltipSimple/TooltipSimple.js

```diff
@@ -17,18 +17,33 @@ const TooltipSimple = ({
 
   const tooltipWrapperClasses = classNames(`bx--tooltip--simple`, className);
   return (
-    <div className={tooltipWrapperClasses}>
-      {children}
-      <div
-        className={tooltipClasses}
-        data-tooltip-text={text}
-        tabIndex="0"
-        role="button"
-        {...other}>
-        {showIcon && (
-          <Icon role="img" name={iconName} description={iconDescription} />
-        )}
-      </div>
+    <div>
+      {showIcon ? (
+        <div className={tooltipWrapperClasses}>
+          {children}
+          <div
+            className={tooltipClasses}
+            data-tooltip-text={text}
+            tabIndex="0"
+            role="button"
+            {...other}>
+            {showIcon && (
```

Comment:
Seems that `showIcon` will never be `false` by the change...?

Reply:
yes  this piece has to be removed since the condition is already verifying and `showIcon` is already true